pkg/api/aim/api/response/run.go

```diff
@@ -19,6 +21,8 @@ import (
 	"github.com/G-Research/fasttrackml/pkg/api/aim/dao/models"
 	"github.com/G-Research/fasttrackml/pkg/api/aim/dao/repositories"
 	"github.com/G-Research/fasttrackml/pkg/api/aim/encoding"
+	mlflowCommon "github.com/G-Research/fasttrackml/pkg/api/mlflow/common"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/services/artifact/storage"
```

Comment:
I'd say we shouldn't introduce a new `aim` -> `mlflow` dependency, since we've managed to get rid of these all pretty recently. Maybe introduce semi-duplicate stuff on the aim side, or move from `mlflow` to `common`? Duplicates for Aim is what I was doing for image search, since the needs are kind of different here.

Reply:
I'll create a ticket for that